guide/payments/request.md

```diff
@@ -0,0 +1,477 @@
+
+### Amount
+
+When generating a single-use payment request, the receiver may want to add an amount to be paid. Entering an amount from the home screen can speed up the requesting flow. Alternatively, users can add an amount to an already generated payment request. Entering an amount needs to be clear to prevent users entering incorrect amounts.
+
+Amount units need to be toggleable between bitcoin, satoshi and local currency values as senders may request in a set unit. Ensure the unit being entered is prominent to prevent entering of the wrong unit amount. More on the [Units & Symbols page]({{ "/guide/payments/units-and-symbols/" | relative_url }}).
+
+### Address types
+
+There are several address types that can be used for on-chain transfer requests. Each address type has unique benefits, such as SegWit saving on network fees. Supporting every address type makes a wallet more interoperable and lets users access the benefits offered by each type. More on the [address types]({{ "/guide/glossary/address/" | relative_url }}) page.
+
+### Meta data
+
+A payment request with only standard data like an amount and date communicates little to both parties about the payment. Users should have the option to attach a note for record-keeping and for the sender to read. Adding tags, labels and any other metadata that may be relevant to give more context to the payment is also beneficial.
+
+#### Invoice expirations
+
+A piece of meta data unique to invoices is an expiration time. This is how long until a single-use invoice expires. By default this is 60 minutes. When an invoice is generated an expiration timer needs to be shown to users alongside the payment request. For more custom use cases, allowing users to adjust this expiration time manually.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/Home.png"
+   retina = "/assets/images/guide/payments/request/Home@2x.png"
+   layout = "shadow"
+   caption = "Home screens can speed up the generating of a single-use payment request."
+   alt-text = "Screen showing home screen number pad entry."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/single-use-payment-request.png"
+   retina = "/assets/images/guide/payments/request/single-use-payment-request@2x.png"
+   layout = "shadow"
+   caption = "A newly generated single-use payment request."
```

Comment:
I'm not convinced about the payment vs. transaction dichotomy. My thoughts are expressed [in much more detail here](https://github.com/BitcoinDesign/Guide/issues/612#issuecomment-1013385394).

I see this dichotomy expressed in the images. Perhaps this is just a nit...it doesn't really affect the content of the page and we can always spend more time discussing this idea, then update images with a more consensus approach.
  
What would be a good distinction?

- Lightning vs. Bitcoin
- Payment vs. Transaction
- Off-chain vs. On-chain
- Instant vs. Slow
- Something else?

Reply:
Went with on-chain / Lightning for now - maybe we can figure something better out and come back to this.